src/App.tsx

```diff
@@ -15,49 +16,139 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  alphabet = 'alphabet',
+  length = 'length',
+  reverse = 'reverse',
+}
```

Comment:
```suggestion
enum SortType {
  Alphabet = 'alphabet',
  Length = 'length',
}
```
Use CamelCase for enum keys
Also, reverse is not one of sort type, same as `alphabet` and `length` - you can select `alphabet` **and** reverse